client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import { styled } from '@mui/material/styles';
+import useStyles from './useStyles';
+import AuthPageWrapper from '../../../components/AuthPageWrapper/AuthPageWrapper';
+import { Typography } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+
+const Input = styled('input')({
+  display: 'none',
+});
+interface ProfilePhotoProps {
+  header: string;
+}
+
+const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header }) => {
+  const classes = useStyles();
+  return (
+    <>
+      <SettingHeader header={header} />
+      <AuthPageWrapper header="Profile Photo">
```

Comment:
How come you are using the AuthPageWrapper? This has nothing to do with an `AuthPage` 

As mentioned in my other comment, the Settings page already handles the outer wrapper so we just need to focus on the inner content.

Reply:
This is why I am seeing duplicate headers.

Reply:
AuthPageWrapper removed. Thanks